### DISCUSSION TITLE
Range filter with number input

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/zag#1448] [Ideas]
An Airbnb like price range filter. It can be controlled with range slider and number input at the same time.

I think it would be fit really nice with a zag machine. Maybe an example integrating range slider with number input.

![image](https://github.com/chakra-ui/zag/assets/53458125/abe4abb7-ba79-4f4e-80ff-f71aadd96a2b)

**Comment by segunadebayo** [comment]
This can be achieved by composing the slider and number input machines.

Here's an example composition:
https://github.com/chakra-ui/zag/blob/main/examples/next-ts/pages/composition/slider-number-input.tsx

It probably needs more fine-tuning, but it's a great starting point.

**Reply by juliomuhlbauer** [reply]
Nice! Thanks a lot!
Will try to implement in Ark too!
